spec.md

```diff
@@ -194,6 +195,11 @@ as a string.
   `String`, `Binary`, `Map`, `URI-reference` or `Timestamp`. The type system is
   intentionally abstract, and therefore it is left to implementations how to
   represent the `Any` type.
+- `Any-context` - A variant type which can take the shapes allowed in a context
```

Comment:
I'm not sure I understand the purpose or value of this. If it's really only for extensions, and we treat unknown extensions as strings, why do we need this?

Reply:
I would leave `Any` untouched. Stating that the Map type can't be used for context attributes is sufficient. `Any` is not a submarine through which you can smuggle `Map` undetected.